lib/marked-renderer.js

```diff
@@ -1,7 +1,22 @@
 const { marked } = require('marked')
 
-/** A custom renderer that avoids wrapping `<img>` tags in `<p>` tags */
+/**
+ * Custom markdown renderer
+ */
 class DesignSystemRenderer extends marked.Renderer {
+  /**
+   * Assume HTML when no code block language provided
+   * (for example, HTML code inside Markdown)
+   */
+  code (code, language, isEscaped) {
+    return !language
+      ? super.html(code)
```

Comment:
One of the side effects of enabling line breaks again in [Marked](https://marked.js.org) is that nested HTML in Markdown now gets handled as "code" when it follows another line break

Since nested HTML lacks a syntax `language` we can pass it straight to `super.html()`

Code blocks (with default or custom language) continue to be wrapped with `<pre><code>`

## Before

<img width="843" alt="Line breaks, before" src="https://user-images.githubusercontent.com/415517/220466638-ba7184e6-9ed0-44bf-b122-5edf3e7fddac.png">

## After

<img width="842" alt="Line breaks, after" src="https://user-images.githubusercontent.com/415517/220466650-e1c4b3d7-3a02-432d-8c0f-24bf618601b7.png">